charts/backingservices/charts/constellation-messaging/values.yaml

```diff
@@ -0,0 +1,28 @@
+---
+# Default values for c11n-messaging-service charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  imageCredentials:
+    registry: "YOUR_DOCKER_REGISTRY"
+    username: "YOUR_DOCKER_REGISTRY_USERNAME"
+    password: "YOUR_DOCKER_REGISTRY_PASSWORD"
+
+enabled: true
+deploymentName: c11n-messaging
+
+# c11n-messaging-service configuration
+c11nMessagingRuntime:
+  replicaCount: 2
+  # docker image of the c11n-messaging-service, example : constellation-messaging/docker-image:dockerTag
+  c11nMessagingImage: "YOUR_C11N_MESSAGING_IMAGE:TAG"
```

Comment:
We need to document where to get the image here: https://github.com/pegasystems/pega-helm-charts#downloading-docker-images-for-your-deployment

Reply:
Added the necessary info.

Reply:
@anuds20 - where is the new information? I don't see your latest changes to the readme. I have to transfer these details to this client-facing documentation >>https://docs.pega.com/client-managed-cloud/87/using-pega-provided-docker-images